pkg/compactor/block_upload.go

```diff
@@ -74,7 +77,7 @@ func (c *MultitenantCompactor) StartBlockUpload(w http.ResponseWriter, r *http.R
 				statusCode: http.StatusRequestEntityTooLarge,
 			}
 		}
-		writeBlockUploadError(err, op, "", logger, w)
+		writeBlockUploadError(err, "", logger, w)
```

Comment:
is there a particular reason why some have no message?

i just imagine someone seeing the log line produced by this statement and then not knowing which of the statements that have no message has produced it. I know that they could go by the line of code, but it would be nice if each log producing line could have something that uniquely identifies it to make it easy to lookup where in the code the message was produced.